docs/docs/security.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Securing Pomerium
+description: >-
+  This page is an index for security-related pages throughout Pomerium's documentation.
+---
+
+# Securing Pomerium
+
+Pomerium is a tool for securing your infrastructure, following the principles of [Zero Trust](/docs/background.md#zero-trust). But that doesn't mean that your stack is "secure" right out of the box. Additionally, security is a battle of give and take; more security often comes at the cost of more complexity, both for the administrator and the end-user. What layers of security you choose to apply (and how you configure them) is highly independent on each use case.
```

Comment:
```suggestion
Pomerium is a tool for securing your infrastructure while adhering to the principles of [Zero Trust](/docs/background.md#zero-trust). But that doesn't mean that your stack is "secure" right out of the box. Additionally, security is a battle of give and take; more security often comes at the cost of more complexity, both for the administrator and the end-user. What layers of security you choose to apply (and how you configure them) is highly independent on each use case.
```

The original comma threw me off and here's my hack, but it might make it a bit more of a complicated sentence. Open to other revisions.